corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/content.less

```diff
@@ -73,6 +73,50 @@ body {
   height: 100%;
 }
 
+.marker-pin {
+  width: 30px;
+  height: 30px;
+  border-radius: 50% 50% 50% 0;
+  background: #c30b82;
```

Comment:
Could this be a standard color? Maybe `@cc-att-pos-mid`, we use the in web apps to draw attention.